### PR TITLE
[fix bug 1237875] Redirect /community/forums/ to /about/forums/

### PR DESCRIPTION
**Pull request by alexgibson**

### CODE REVIEW

**Comment by alexgibson** [comment]
@pmclanahan r?

File: bedrock/mozorg/redirects.py [hunk 1]
```diff
@@ -246,6 +246,9 @@ def decider(request, **kwargs):
     # to force it to match all repeating slashes in one go.
     redirect(r'^(.*[^/])//+([^/].*)$', '/{}/{}', locale_prefix=False),
 
+    # bug 1237875
+    redirect(r'^community/forums/?$', 'mozorg.about.forums.forums'),
+
     # bug 927442
     redirect(r'^(firefox/)?community/?', 'mozorg.contribute'),
```
Comment (alexgibson) [inline bedrock/mozorg/redirects.py]: Note: I had to add the redirect above this line, which was causing the URL to redirect to the `/contribute` page.

File: bedrock/mozorg/redirects.py [hunk 2]
```diff
@@ -578,10 +581,10 @@ def decider(request, **kwargs):
     # bug 1233015
     redirect(r'^about/partnerships/contentservices(/.*)?$', 'mozorg.partnerships'),
 
+    # Bug 1235853
+    redirect(r'^facebookapps(/.*)?$', 'firefox.new'),
+
     # bug 832348 **/index.html -> **/
     # leave this at the bottom
```
Comment (alexgibson) [inline bedrock/mozorg/redirects.py]: Note: I'm also moving the `facebookapps` redirect above this line, as I only just noticed this comment.

File: test_redirects/map_globalconf.py [hunk 3]
```diff
@@ -938,5 +938,8 @@
 
     # bug 1238248
     url_test('/firefox/push/',
-             'https://support.mozilla.org/kb/push-notifications-firefox')
+             'https://support.mozilla.org/kb/push-notifications-firefox'),
+
+    # bug 1237875
+    url_test('/community/forums/', '/about/forums/')
```
Comment (pmclanahan) [inline test_redirects/map_globalconf.py]: NIT: Always best to add the comma at the end for when we need to add the next one. Not a requirement, just FYI.

**Comment by pmclanahan** [comment]
r+

File: test_redirects/map_globalconf.py (hunk 3 above)
Comment (alexgibson) [inline test_redirects/map_globalconf.py]: Added a comma, thanks

**Pull request merged**